addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -42,9 +60,17 @@
                 </div>
 
                 <div class="o_control_panel_navigation d-flex flex-wrap flex-md-nowrap justify-content-end gap-3 gap-lg-1 gap-xl-3 order-1 order-lg-2 flex-grow-1">
+                    <div t-if="state.showTopBar &amp;&amp; state.topBarActions?.length > 0" class="d-flex d-md-none">
+                        <t t-call="web.topBarActionsDropdown" />
+                    </div>
                     <div t-if="pagerProps and pagerProps.total > 0" class="o_cp_pager text-nowrap " role="search">
                         <Pager t-props="pagerProps"/>
                     </div>
+                    <button t-if="state.topBarActions?.length > 0" class="btn btn-secondary"
+                            t-attf-class="{{state.showTopBar ? 'active' : ''}}"
+                            t-on-click="() => state.showTopBar = !state.showTopBar">
+                        <i class="fa fa-eye" />
```

Comment:
Btw, I don't think an eye icon represents the idea of the top bar really well :thinking:

Reply:
Hum I could agree, but I don't know which icon could suit better, TBD with Alex I guess

Reply:
I guess we'll see in testing

Reply:
yep